### DISCUSSION TITLE
How to use cudnn or cublas by DPC++?

### DISCUSSION THREAD

**Discussion by malixian** [intel/llvm#3712] [Q&A]
Is it possible to achieve this demand? Can you give me an example？

**Comment by alexbatashev** [comment]
Both [oneDNN](https://github.com/oneapi-src/oneDNN) and [oneMKL](https://github.com/oneapi-src/oneMKL) have CUDA backend, and as far as I know, they use cudnn and cublas under the hood. As a bonus you get full DPC++ compatibility and performance portability across multiple vendors and device types. Would that fit your needs?
